test/functional/data.spec.js

```diff
@@ -60,10 +61,23 @@ describe('swag-for-dev', function () {
 						});
 					}
 
-					it(opportunity.name + ' has a valid image', function () {
+					it(opportunity.name + ' has a valid image', function (done) {
 						this.timeout(10000);
 						this.slow(1500);
-						return checkURL(opportunity.image);
+						got(opportunity.image)
+							.then(function (response) {
+								sharp(response.rawBody, {limitInputPixels: false})
+									.toBuffer()
+									.then(function () {
+										done();
+									})
+									.catch(function (err) {
+										done(err);
+									});
+							})
+							.catch(function (err) {
+								done(err);
+							});
```

Comment:
```suggestion
						const {rawBody} = await got(opportunity.image)
						await sharp(response.rawBody, {limitInputPixels: false}).toBuffer()
```

Reply:
To maintain homogeneity with previous code, I used Promise instead of async/await.